lib/src/client.ts

```diff
@@ -169,6 +170,54 @@ export class AsgardeoAuthClient<T> {
         return AsgardeoAuthClient._instanceID;
     }
 
+    /**
+   * This is an async method that returns a Promise that resolves with the authorization URL parameters.
+   *
+   * @param config - (Optional) A config object to force initialization and pass
+   * custom path parameters such as the fidp parameter.
+   * @param userID - (Optional) A unique ID of the user to be authenticated. This is useful in multi-user
+   * scenarios where each user should be uniquely identified.
+   *
+   * @returns - A promise that resolves with the authorization URL parameters.
+   *
+   * @example
+   * ```
+   * auth.getAuthorizationURLParams().then((params)=>{
+   *  // console.log(params);
+   * }).catch((error)=>{
+   *  // console.error(error);
+   * });
+   * ```
+   *
+   * {@link https://github.com/asgardeo/asgardeo-auth-js-sdk/tree/master#getAuthorizationURLParams}
+   *
+   * @preserve
```

Comment:
What is the usage of this doc tag here?

Reply:
@link tag is providing a link to the documentation. Refer [docs.](https://jsdoc.app/tags-inline-link)

@preserve tag is a custom tag defined. Refer [tsdoc.json](https://github.com/asgardeo/asgardeo-auth-js-core/blob/6bb00d5a478f532da3261d24e625131ca6b7ccec/lib/tsdoc.json#L5)